tests/model_hub_tests/tf_hugging_face_tests/precommit_models

```diff
@@ -0,0 +1 @@
+google/vit-base-patch16-224
```

Comment:
not needed, we should have the common precommit files that can contain different sorts of models from TF Hub, S3D share and HF

Reply:
Done.